.github/.gitignore

```diff
@@ -1 +0,0 @@
-*.html
```

Comment:
I have no idea. `git log --follow -p .github/.gitignore` reveals that this has been around since the 1st real commit of the repo, I assume it was copy pasted from the old stress testing repo

Reply:
However if you think this is unneeded, let us remove this?

Reply:
Please disregard -- sorry. I now realize the entire .gitignore is deleted along with other files in .github. I first though you had specifically removed the line where we ignore html files.